src/js/validator.js

```diff
@@ -205,13 +205,13 @@ export const LOCATION_SCHEMA = {
         },
       },
     },
-    "gsx$noidneeded": {
-      "type": "object",
-      "properties": {
-        "$t": {
-          "type": "string"
-        }
-      }
+    "gsx$noidneeded": {	
```

Comment:
Since this transitions to Airtable, are the Google Spreadsheet validators still needed?

Reply:
Yeah, no probably not. Actually the validator isn't really being used at all in the rework but I didn't want to delete the whole file in case we did want to actually use it, but I can remove the google validators